doc/man/8/radosgw-admin.rst

```diff
@@ -260,9 +459,9 @@ Options
 
    The e-mail address of the user.
 
-.. option:: --display-name=name
```

Comment:
can you move it after --access as it is in the help command?

Reply:
Fixed